crates/biome_js_analyze/src/lint/nursery/no_import_cycles.rs

```diff
@@ -35,8 +35,8 @@ declare_lint_rule! {
     /// ### Invalid
     ///
     /// **`foobar.js`**
-    /// ```js
-    /// import { baz } from "./baz.js";
+    /// ```js,expect_diagnostic,file=foobar.js
```

Comment:
We could read this `file` tag in our codegen in website to show the filename on the snippet. This can be a follow-up improvement :)

e.g.:

<img width="776" height="135" alt="Image" src="https://github.com/user-attachments/assets/1752d4aa-2c07-4b1a-ad4c-92791d51b96c" />

Reply:
that sounds like a cool idea. will definitely look into it